spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/DefaultAmqpHeaderMapper.java

```diff
@@ -70,11 +70,13 @@
 		STANDARD_HEADER_NAMES.add(AmqpHeaders.CONTENT_LENGTH);
 		STANDARD_HEADER_NAMES.add(AmqpHeaders.CONTENT_TYPE);
 		STANDARD_HEADER_NAMES.add(AmqpHeaders.CORRELATION_ID);
+		STANDARD_HEADER_NAMES.add(AmqpHeaders.DELAY);
```

Comment:
I think we should mention these new headers for mapping in the `AMQP Message Headers` doc.
And short expression for Delayed Exchanges in the What's New would be good noise as well :smile: